content/docs/5.1/concepts/tikv-in-5-minutes.md

```diff
@@ -30,139 +32,143 @@ The following steps are only for test environments. For production environments
 
-    Redeclare the global environment variables:
+    1. Redeclare the global environment variables:
 
-    ```bash
-    source .bash_profile
-    ```
+        ```bash
+        source .bash_profile
+        ```
 
-    Confirm whether TiUP is installed:
+    2. Confirm whether TiUP is installed:
 
-    ```bash
-    tiup
-    ```
+        ```bash
+        tiup
+        ```
+
+    <!-- Explain the results -->
 
-3. If TiUP is already installed, update the TiUP playground component to the latest version:
+3. If TiUP is already installed, update the TiUP Playground component to the latest version:
 
     ```bash
     tiup update --self && tiup update playground
     ```
 
-4. Use TiUP playground to start a local TiKV cluster
-
-    Show TiUP version:
+4. Use TiUP Playground to start a local TiKV cluster. Before you do that, check your TiUP version using the following command:
 
     ```bash
     tiup -v
     ```
 
-    version >= 1.5.0:
+    - If your TiUP version is v1.5.0 or later, execute the following command to start a local TiKV cluster:
 
-    ```bash
-    tiup playground --mode tikv-slim
-    ```
+        ```bash
+        tiup playground --mode tikv-slim
+        ```
 
-    version < 1.5.0:
+    - If your TiUP version is earlier than v1.5.0, execute the following command to start a local TiKV cluster:
 
-    ```bash
-    tiup playground
-    ```
+        ```bash
+        tiup playground
+        ```
 
     {{< info >}}
-Refer to [TiUP playground document](https://docs.pingcap.com/tidb/stable/tiup-playground) to find more TiUP playground commands.
+Refer to [TiUP playground document](https://docs.pingcap.com/tidb/stable/tiup-playground) for more TiUP Playground commands.
     {{< /info >}}
 
-## Step 2: Monitor the cluster
-
-Open [http://127.0.0.1:3000](http://127.0.0.1:3000) and login to the Grafana (username: admin password: admin).
+## Monitor the TiKV cluster
 
-Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
+After the TiKV cluster is started using TiUP Playground, you might need to monitor the cluster. Perform the following steps to see the needed monitoring metrics:
 
-## Step 3: Hello, World!
+1. Open your browser, access [http://127.0.0.1:3000](http://127.0.0.1:3000), and login to the Grafana Dashboard. By default, the username is `admin` and the password is `admin`.
 
-### Java
+2. Open the `TiKV-Summary` page on the Grafana Dashboard and find the monitoring metrics of your TiKV cluster.
 
-1. Download jars
+## Write data to and read data from the TiKV cluster
 
-    ```bash
-    curl -o tikv-client-java.jar https://download.pingcap.org/tikv-client-java-3.2.0-SNAPSHOT.jar
-    curl -o slf4j-api.jar https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.16/slf4j-api-1.7.16.jar
-    ```
+To write to and read from the TiKV cluster, you can use either Java or Python script. In the following two examples, Java and Python are respectively used to write "Hello World!" to TiKV.
```

Comment:
@marsishandsome Are Python and Java the only two ways to write and read?

Reply:
There will be another language APIs, e.g. go c++. But it's not ready now.

Reply:
```suggestion
To write to and read from the TiKV cluster, you can use either Java, Go, Rust, C or Python script. In the following two examples, Java and Python are respectively used to write "Hello World!" to TiKV.
```